docs/content/latest/deploy/checklist.md

```diff
@@ -87,7 +87,32 @@ It can also leverage multiple disks per node and has been tested beyond 10 TB of
 
 Write-heavy applications usually require more disk IOPS (especially if the size of each record is larger), therefore in this case the total IOPS that a disk can support matters. On the read side, if the data does not fit into the cache and data needs to be read from the disk in order to satisfy queries, the disk performance (latency and IOPS) will start to matter.
 
-YugabyteDB uses per-tablet [size tiered compaction](../../architecture/concepts/yb-tserver/). Therefore the typical space amplification in YugabyteDB tends to be in the 10-20% range.
+#### Storage overprovisioning
+
+YugabyteDB stores data compressed by default. We have seen 3-5x compression in a lot of workloads. In some workloads
```

Comment:
I feel this needs to be split into two parts:

## Add to this section only the following:

YugabyteDB stores data compressed by default. The effectiveness of compression depends on the data set. For example, if the data has already been compressed, then the additional compression at the storage layer of YugabyteDB will not be very effective.

Plan for about 20% head room on each node to allow space for miscellaneous overheads such as temporary additional space needed for compactions, metadata overheads, etc.

## Separate section
The discussion on disk space and fault tolerance must be explained in a separate section in a lot more detail. cc @schoudhury @stevebang to think about where to put this. Also, the description below is too terse for anyone to really follow.